src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -692,6 +814,21 @@ public ListBoxModel doFillConsoleOutputLinkStyleItems() {
         }
     }
 
+    /**
+     * Because java 6
+     *
+     */
+    public interface Function<F, T> {
+        /**
+         *
+         * @param input
+         *  an input
+         * @return
+         *  a result
+         */
+        T apply(F input);

```

Comment:
Updated - this is a JDK8 function, but we're not requiring our users to have JDK8, so I copied it